x/escrow/time_test.go

```diff
@@ -22,3 +22,23 @@ func TestIsExpired(t *testing.T) {
 		t.Error("past is not expired")
 	}
 }
+
+func TestIsExpiredRequiresBlockTime(t *testing.T) {
+	hasPanic := make(chan bool)
+
+	go func() {
```

Comment:
why a go routine here?
a simple function, with defer block like it is, and with the t.Fatal() following the isExpired call would do the same job and be simpler and clearer...

Oh, did I mention that testify has `assert.Panics(func () {...})`. :laughing:

Reply:
Done in https://github.com/iov-one/weave/pull/409/commits/674aa58939345a06154a5b91897f1f4d34fe8bd0